bin/interpreter.sh

```diff
@@ -296,5 +296,6 @@ else
 fi
 
 # Don't remove this echo, it is for diagnose, this line of output will be redirected to java log4j output.
-echo "Interpreter launch command: ${INTERPRETER_RUN_COMMAND[@]}"
+# Output that starts with `[INFO]` will be redirected to log4j output.
```

Comment:
Isn't it better to express "log4j INFO output" or "log4j INFO message" more precisely than "log4j output"? Depending on the log level setting, debug messages can also be displayed in the output.

Reply:
Thanks for the comment @cuspymd It is fixed now.